app/models/concerns/draft.rb

```diff
@@ -0,0 +1,22 @@
+module Draft
```

Comment:
I think you can use https://github.com/brainspec/enumerize instead of creating separate concern for statuses